app/graphql/hooks.ts

```diff
@@ -81,13 +85,7 @@ export const makeUseQuery =
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [queryKey, options.pause]);
 
-    const { data, ...rest } = result;
-    const res = {
-      ...rest,
-      data: result.queryKey === queryKey || keepPreviousData ? data : null,
-    };
-
-    return [res, executeQuery] as const;
+    return [result, executeQuery] as const;
```

Comment:
Looks like a tricky bug, is there an easy way to add a test for it, I see we already tests in hooks.spec.tsx, maybe it'd be easy, not sure.